sdk/schemaregistry/schema-registry-json/test/public/errors.spec.ts

```diff
@@ -224,19 +213,34 @@ describe("Error scenarios", function () {
           44, 34, 110, 97, 109, 101, 34, 58, 34, 120, 34, 125,
         ])
       );
-      serializedValue.data = Buffer.from([
+      // const nodeMessage = /Unexpected end of JSON input/;
+      serializedValue.data = Uint8Array.from([
         123, 34, 102, 97, 118, 111, 114, 105, 116, 101, 78, 117, 109, 98, 101, 114,
       ]);
-      await assertError(serializer.deserialize(serializedValue), {
-        causeMessage: /Unexpected end of JSON input/,
-      });
-      serializedValue.data = Buffer.from([
+      if (isNode){
+        await assertError(serializer.deserialize(serializedValue), {
+          causeMessage: /Unexpected end of JSON input/,
+        });
+      } else {
+        await assertError(serializer.deserialize(serializedValue), {
+          causeMessage: /Unterminated string in JSON at position/,
+        });
+      }
```

Comment:
perhaps we can move the `isNode` check to inside the `causeMessage` definition so we don't have to duplicate the deserialize call?